src/app/core/core.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ButtonComponent } from './button/button.component';
+import { LogoComponent } from './logo/logo.component';
+import { HeaderComponent } from './header/header.component';
+import { FooterComponent } from './footer/footer.component';
+import { InputComponent } from './input/input.component';
+
+
+
+
+@NgModule({
+  declarations: [ButtonComponent, LogoComponent, HeaderComponent, FooterComponent, InputComponent],
+  imports: [
+    CommonModule,
+  ],
+  exports: [
+    LogoComponent,
+    ButtonComponent,
+    HeaderComponent,
+    FooterComponent,
+    InputComponent,
+  ]
+})
+export class CoreModule { }
```

Comment:
It looks like a shared module but not core. Please rename it to avoid misleading in the future.

Usefull links:
- [Confusing Information on SharedModule and CoreModule](https://github.com/johnpapa/angular-styleguide/issues/855)
- [Angular best practices: CoreModule vs SharedModule](https://medium.com/@benmohamehdi/angular-best-practices-coremodule-vs-sharedmodule-25f6721aa2ef)
- [Angular2: CoreModule vs SharedModule](https://stackoverflow.com/questions/42695931/angular2-coremodule-vs-sharedmodule)
- [Angular - Core and Shared Modules](https://frontpills.com/posts/2019/core-shared-modules/)
- [Sharing Modules](https://angular.io/guide/sharing-ngmodules)